### PR TITLE
Added IsNone() and GetPyNone()

### PR DESCRIPTION
**Pull request by henon**
### What does this implement/fix? Explain your changes.

IsNone() and GetPyNone() were needed for Numpy.NET to be able to find out if a PyObject is none. 

### Does this close any currently open issues?

No.

### Any other comments?

These are in use for a long time now, I just now figured out how to PR them. I'll commit the necessary things from the checklist below.

### Checklist

Check all those that are applicable and complete.

-   [ ] Make sure to include one or more tests for your change
-   [ ] If an enhancement PR, please create docs and at best an example
-   [x] Add yourself to [`AUTHORS`](../blob/master/AUTHORS.md)
-   [x] Updated the [`CHANGELOG`](../blob/master/CHANGELOG.md)

### CODE REVIEW

**Comment by henon** [comment]
I have no idea how to add a test and the change is so small that it doesn't really need documentation.

**Comment by filmor** [comment]
I completely get the `IsNone`, but what do you need the `GetPyNone` for? It's a bit arbitrary here, isn't it?

**Comment by lostmsu** [comment]
@filmor re:`GetPyNone` it is actually useful. I have it in my branch too. That's the easiest way to pass `None`.

Though as a static property (and I don't see why it should not be a property).

File: src/runtime/runtime.cs [hunk 1]
```diff
@@ -473,6 +473,19 @@ private static void ResetPyMembers()
         internal static IntPtr PyNone;
         internal static IntPtr Error;
 
+        public static PyObject _none;
+        public static PyObject GetPyNone()
```
Comment (lostmsu) [inline src/runtime/runtime.cs:477]: A property would seem more natural.

File: src/runtime/runtime.cs [hunk 2]
```diff
@@ -473,6 +473,19 @@ private static void ResetPyMembers()
...
+        {
+            if (_none == null)
+            {
+                var result = Runtime.PyNone;
+                Runtime.XIncref(result);
+                _none= new PyObject(result);
```
Comment (lostmsu) [inline src/runtime/runtime.cs:483]: I do not think there's much point in caching result.

But if you do, you must deal with:
1. When runtime is shutdown and restarted, this value should be reinitialized.
2. On the .NET side writing to `_none` must be properly synchronized. E.g. you need a lock or an atomic swap.

Again, however, I think you can just not cache the value.

Comment (henon) [reply src/runtime/runtime.cs:483]: ok, I best remove the caching

**Comment by henon** [comment]
You might wanna squash this PR into a single commit.

**Comment by filmor** [comment]
Thank you :)

**Pull request merged**